core/src/main/res/values/strings.xml

```diff
@@ -145,6 +146,11 @@
     <string name="download_request_error_dialog_message_prefix">An error occurred when trying to download the file:\u0020</string>
     <string name="authentication_notification_title">Authentication required</string>
     <string name="authentication_notification_msg">The resource you requested requires a username and a password</string>
+    <string name="confirm_mobile_download_dialog_title">Confirm Mobile Download</string>
+    <string name="confirm_mobile_download_dialog_message_not_in_queue">Downloading over mobile data connection disable in settings.\n\nEnable temporarily or just add to queue?\n\n<small>Your choice will be remember for 10 minutes.</small></string>
```

Comment:
Is there a typo in both lines? 
" connection disable in..." should be 
" connection (is) disabled in..."?